src/main/java/framework/components/Header.java

```diff
@@ -0,0 +1,42 @@
+package framework.components;
+
+import framework.helpers.WebDriverWaitHelper;
+import framework.pages.AbstractPage;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class Header {
```

Comment:
It's better to create BasePage class and move this component and methods there.

Reply:
Done.